lib/order.rb

```diff
@@ -0,0 +1,78 @@
+require 'csv'
+
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id,products,customer,fulfillment_status=:pending)
+    @id = id
+    @products = products
+    @customer = customer
+    
+
+    @fulfillment_status = fulfillment_status
+    unless [:pending,:paid,:processing,:shipped,:complete].include?(@fulfillment_status)
```

Comment:
Consider creating a constant variable at the start of your script `VALID_STATUS = [:pending,:paid,:processing,:shipped,:complete]`